### PR TITLE
default to not-normalized

### PR DESCRIPTION
**Pull request by urbanfly**
When the component loads, there is only a single location and the data is not normalized.

When a 2nd location is selected, normalization is automatically enabled.

When the 2nd location is removed, normalization is disabled.

Fixes issue #1471

### CODE REVIEW

File: src/components/Explore/Explore.tsx [hunk 1]
```diff
@@ -64,8 +64,14 @@ const Explore: React.FunctionComponent<{
   ]);
 
   const onChangeSelectedLocations = (newLocations: Location[]) => {
+    const selectedLocations = uniq([currentLocation, ...newLocations]);
```
Comment (pnavarrc) [inline src/components/Explore/Explore.tsx:67]: Consider renaming this variable so it has a different name than the state variable `selectedLocations`. I know that the local variable will take precedence, but a different name will make that more obvious.

**Review by pnavarrc — CHANGES_REQUESTED** [review]
Hi @urbanfly, thanks for your contribution! It looks generally good, except that it overwrites the user preference if they have multiple locations and they don't want to normalize. For example, if I have Kansas and Washington enabled, and then uncheck normalization, and then add Texas, it will enable normalization back again, overwriting my preference. To summarize:
- The default value for a single location should be not normalized
- The default value for more than one location should be normalized
- The normalization setting should not change if the user adds more locations, unless when changing from one to two or from two to one.

Let me know if that makes sense and if you need help!

**Comment by urbanfly** [comment]
Pinging @pnavarrc @chasulin since they are the authors of the related PRs #1406 and #1433 

Pinging @mikelehen since he is the approver of the related PR.

**Comment by urbanfly** [comment]
@pnavarrc Thanks for the feedback. I understand and I've made the adjustments.

**Review by pnavarrc — APPROVED** [review]
Looks good, thanks @urbanfly !

**Pull request merged**